Shapes/shapes/v2/datatype_map.ttl

```diff
@@ -0,0 +1,139 @@
+###############################################################################
+# RML Datatype Map and datatype shortcut shape                                #
+# Copyright Dylan Van Assche, IDLab - UGent - imec (2023)                     #
+###############################################################################
+@prefix sh: <http://www.w3.org/ns/shacl#> .
+@prefix schema: <http://schema.org/> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rml: <http://w3id.org/rml/core/> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> . 
+
+schema:RMLDatatypeMapShape
+    a sh:NodeShape ;
+    sh:ignoredProperties (rdf:type) ;
+    sh:name "rml:datatypeMap" ;
+    sh:description """
+    Represents a Datatype Map.
+    """ ;
+    sh:message """
+    rml:DatatypeMap must specify an rml:template, rml:reference, or rml:constant
+    with the IRI of the datatype.
+    """ ;
+
+    # Exactly one rml:template or one rml:reference or one rml:constant is
+    # required
+    sh:property [
```

Comment:
I think it would be easier if we give this property shape a URI.

I think that the DatatypeMap is an ExpressionMap so in principle its constraints should follow the constraints of the Expression Map.

Reply:
There's no inheritance on purpose to keep things readable and understandable which is also why I never use IRIs.